### DISCUSSION TITLE
Missing placeholder copy in "Add secret" modal

### DISCUSSION THREAD

**Discussion by GioAc96** [mia-platform/community#236] [Q&A] [answered]
### Which section?

Console

### Which is the version?

Mia Platform: 9.5.3, Console: 1.73.9

### Description

When opening the "Add secret" modal for a microservice, the placeholder for the "Configuration name" field is missing the appropriate copy and defaulting to the message id.

### Actual Outcome

![image](https://user-images.githubusercontent.com/21307773/233343837-5fe43599-15c7-498c-b396-6f984a5dbd60.png)

**Comment by davidebianchi** [comment, marked as answer]
This is resolved in newer version of Console, from the v10. Can you upgrade it?
